includes/class-boldgrid-inspirations-dashboard.php

```diff
@@ -116,18 +116,72 @@ public function add_hooks() {
 		}
 	}
 
+	/**
+	 * Add Onboarding Progress Bar to Admin Bar.
+	 *
+	 * @since 2.8.0
+	 */
+	public function add_onboarding_progress() {
+		$config   = Boldgrid_Inspirations_Config::get_format_configs();
+		$progress = get_option( $config['onboarding_progress_option'], false );
+
+		// Verify user is logged in.
+		if ( ! is_user_logged_in() || ! current_user_can( 'manage_options' ) ) {
+			return;
+		}
+
+		if ( false === $progress &&
+			empty( get_option( $config['onboarding_tasks_option'] ) ) &&
+			! empty( get_option( 'boldgrid_install_options' ) ) ) {
+
+			$this->create_onboarding_tasks( get_option( 'boldgrid_install_options' ) );
```

Comment:
Does this mean we only create tasks once, after the user has completed a deployment? If so, what if we wanted to add a new item to the checklist in 2 weeks?